docs/testplans/ras/ras-ingestion-test-plan.md

```diff
@@ -0,0 +1,41 @@
+# Test plan for Gen3's implementation of Researcher Auth Services (RAS)
+
+## One-liner overview of the service
+The Resercher Auth Services (RAS) is a specification from the Global Alliance for Genomics and Health (GA4GH) that defines a common process for Authentication, Authorization and Logging around NIH's open and controlled data assets and repositories. This test plan focuses on the Gen3 implementation of such spec.
```

Comment:
RAS is technically not a spec from GA4GH but rather an implementation of GA4GH Passport/Visa

Reply:
Pushing a commit to amend the overview.
Thx @jiaqi216